api/src/main/java/de/valtech/aecu/api/groovy/console/bindings/ContentUpgrade.java

```diff
@@ -768,5 +770,14 @@ public interface ContentUpgrade {
      */
     void run(boolean dryRun) throws PersistenceException, AecuException;
 
+    /**
+     * Creates a label in the i18n node under the corresponding language node.
```

Comment:
Give more detailed information that a sling:key and sling message is set on a node with name key for example

Reply:
Extended the information in the description